src/test/lang/parser/functions_standalone_instantiate_test.cpp

```diff
@@ -2,7 +2,8 @@
 #include <sstream>
 #include <test/test-models/good-standalone-functions/basic.hpp>
 #include <test/test-models/good-standalone-functions/special_functions.hpp>
-#include <test/test-models/good-standalone-functions/integrate.hpp>
+// integrate_* functions are no longer supported in stanc2 as of Stan 2.24
+// #include <test/test-models/good-standalone-functions/integrate.hpp>
```

Comment:
This was commented out as /lang tests are meant for stanc2 and stanc2 will no longer support 
It does not make a difference for our testing pipeline, except if you wish to run /lang tests for stanc2.